lib/node_modules/@stdlib/stats/base/dvarmpn/src/addon.c

```diff
@@ -35,12 +35,31 @@
 static napi_value addon( napi_env env, napi_callback_info info ) {
 	STDLIB_NAPI_ARGV( env, info, argv, argc, 5 );
 	STDLIB_NAPI_ARGV_INT64( env, N, argv, 0 );
-	STDLIB_NAPI_ARGV_INT64( env, stride, argv, 4 );
+	STDLIB_NAPI_ARGV_INT64( env, strideX, argv, 4 );
 	STDLIB_NAPI_ARGV_DOUBLE( env, mean, argv, 1 );
 	STDLIB_NAPI_ARGV_DOUBLE( env, correction, argv, 2 );
-	STDLIB_NAPI_ARGV_STRIDED_FLOAT64ARRAY( env, X, N, stride, argv, 3 );
-	STDLIB_NAPI_CREATE_DOUBLE( env, stdlib_strided_dvarmpn( N, mean, correction, X, stride ), v );
+	STDLIB_NAPI_ARGV_STRIDED_FLOAT64ARRAY( env, X, N, strideX, argv, 3 );
+	STDLIB_NAPI_CREATE_DOUBLE( env, stdlib_strided_dvarmpn( N, mean, correction, X, strideX ), v );
```

Comment:
```suggestion
	STDLIB_NAPI_CREATE_DOUBLE( env, API_SUFFIX(stdlib_strided_dvarmpn)( N, mean, correction, X, strideX ), v );
```

you'll also need to include `stdlib/blas/base/shared.h`

Reply:
Please recheck whether we need `shared.h` package  is needed here, as it runs without it also